src/main/java/com/rocketden/main/util/Utility.java

```diff
@@ -0,0 +1,24 @@
+package com.rocketden.main.util;
+
+import java.util.Random;
+
+import com.rocketden.main.controller.v1.BaseRestController;
+
+public class Utility {
+
```

Comment:
Thoughts on converting this into a Component that way you can autowire it? Similarly, (if autowiring) thoughts on moving/renaming it to something like GeneratorService?

Reply:
I like the idea of renaming it to `GeneratorService`, that could allow me to keep the name `Utility.java` for the test file. I'll ask the question of how the auto-wiring is done in this context on the call today.

Reply:
Ok, I autowired the `Utility` class -- I didn't change the name to `GeneratorService` because I thought that we could keep adding utility-type methods.

I also made `Utility.java` a `@Component` class so it could be autowired -- is this the proper way of doing this @alankbi?

Reply:
I think that's good - either Component or Service I think are appropriate

Reply:
Ok perfect! -- I'll keep `@Component` then.